build/uno.winui.targets

```diff
@@ -26,6 +26,7 @@
 			<UnoDefineConstants Condition="'$(MSBuildThisFile)'=='uno.winui.targets'">$(UnoDefineConstants);HAS_UNO_WINUI</UnoDefineConstants>
 			<UnoDefineConstants>$(UnoDefineConstants);UNO_HAS_FRAMEWORKELEMENT_MEASUREOVERRIDE</UnoDefineConstants>
 			<UnoDefineConstants>$(UnoDefineConstants);UNO_HAS_NO_IDEPENDENCYOBJECT</UnoDefineConstants>
+			<UnoDefineConstants Condition="'$(TargetFramework)'=='xamarinios10' or '$(TargetFramework)'=='xamarinmac20'">$(UnoDefineConstants);UNO_HAS_UIELEMENT_IMPLICIT_PINNING</UnoDefineConstants>
```

Comment:
UNO_HAS_UIELEMENT_IMPLICIT_PINNING  -> Do we have a table with all those constants and what they mean, when they're used, etc?

Reply:
Done, see Uno.CrossTargetting.props.